### PR TITLE
Go to Rosyn 2.9

### PR DESCRIPTION
**Pull request by NTaylorMullen**
Moved to Roslyn version 2.9.0-beta4-62911-02. This is the version that exists in VS 15.8 preview 2.

Universe PR: https://github.com/aspnet/Universe/pull/1167

### CODE REVIEW

File: src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolverFactory.cs [hunk 1]
```diff
@@ -16,36 +12,10 @@ internal class OOPTagHelperResolverFactory : ILanguageServiceFactory
     {
         public ILanguageService CreateLanguageService(HostLanguageServices languageServices)
         {
-            if (!IsRemoteClientWorking())
```
Comment (NTaylorMullen) [inline src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolverFactory.cs]: @rynowak this was my primary reason for this PR. Figured it made sense to remove but wanted to make sure you didn't have any other meaning behind this bit.

**Comment by NTaylorMullen** [comment]
ping @rynowak

File: src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolverFactory.cs (hunk 1 above)
Comment (rynowak) [reply src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolverFactory.cs]: Yes, this was just a temporary workaround.

File: build/dependencies.props [hunk 2]
```diff
@@ -47,16 +47,16 @@
...
+    <VSIX_MicrosoftCodeAnalysisCSharpPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftCodeAnalysisCSharpPackageVersion>
+    <VSIX_MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>
+    <VSIX_MicrosoftCodeAnalysisEditorFeaturesTextPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftCodeAnalysisEditorFeaturesTextPackageVersion>
+    <VSIX_MicrosoftCodeAnalysisRemoteRazorServiceHubPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftCodeAnalysisRemoteRazorServiceHubPackageVersion>
+    <VSIX_MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>
+    <VSIX_MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>
+    <VSIX_MicrosoftVisualStudioLanguageServicesPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftVisualStudioLanguageServicesPackageVersion>
+    <VSIX_MicrosoftVisualStudioLanguageServicesRazorRemoteClientPackageVersion>2.9.0-beta4-62911-02</VSIX_MicrosoftVisualStudioLanguageServicesRazorRemoteClientPackageVersion>
```
Comment (rynowak) [inline build/dependencies.props]: These changes also need to go to universe btw

Comment (NTaylorMullen) [reply build/dependencies.props]: Yup, already have a PR for that pending

**Pull request merged**